src/condor_startd.V6/VolumeManager.cpp

```diff
@@ -181,7 +181,7 @@ VolumeManager::MountFilesystem(const std::string &device_path, const std::string
         return false;
     }
 
-    if (-1 == chown(mountpoint.c_str(), get_real_condor_uid(), get_real_condor_gid())) {
+    if (-1 == chown(mountpoint.c_str(), get_user_uid(), get_user_gid())) {
         err.pushf("VolumeManager", 14, "Failed to chown the new execute mount to condor user: %s (errno=%d)",
```

Comment:
Looks like this text needs an update?  Maybe something like:

```suggestion
        err.pushf("VolumeManager", 14, "Failed to chown the new execute mount to user: %s (errno=%d)",
```